### ISSUE TITLE
Can I train the model from scratch?

### ISSUE THREAD

**Issue by michaelzfm** [deu30303/RUC#11]
Hi, I found that if I don't use the preetrained model, the code don't work. So I can't train from scratch?  If I need to train with other dataset like Imagenet, or use other backbone like resnet50, how can I find pretrained model?  Thank you.

**Comment by gauravjain10** [comment]
yeah like for example we want to check for dogs and cats classification.so if we gave them as input,it will show type mismatch error
while loading pretrained models weights and bias(cifar10) because they were trained on 10 classes.so what should we do now?

**Comment by deu30303** [comment]
Hi, thank you for your interest in our work!

Our method is add-on module on top of the off-the-shelf clustering model.
So I also borrowed pretrained model from SCAN github.
https://github.com/wvangansbeke/Unsupervised-Classification
If you want to train from scratch, please refer above SCAN github :) 

Thank you again for your interest.

**Comment by chengwuxinlin** [comment]
Hello, I'm wondering if can I just mute 

https://github.com/deu30303/RUC/blob/d056fe713b15f216fde98a2af0cad9e3fc5359a9/main_ruc_cifar10.py#L299

to train from scratch. Thanks
